NEWS

```diff
@@ -11,10 +11,12 @@ To see all issues & pull requests closed by this release see the [Git closed mil
 
 * [#2057](https://github.com/pgRouting/pgrouting/issues/2057): trspViaEdges columns in different order
 * [#2087](https://github.com/pgRouting/pgrouting/issues/2087): pgr_extractVertices to proposed
+* [#2201](https://github.com/pgRouting/pgrouting/issues/2201): pgr_depthFirstSearch and pgr_dijkstraNearCost to proposed
```

Comment:
```suggestion
* [#2201](https://github.com/pgRouting/pgrouting/issues/2201): pgr_depthFirstSearch to proposed
```
`pgr_dijkstraNearCost` was already moved in the last PR (same in `release_notes`)